External/FEXCore/include/FEXCore/Core/CoreState.h

```diff
@@ -132,15 +132,7 @@ namespace FEXCore::Core {
     CPUState() {
       // Initialize default CPU state
       rip = ~0ULL;
```

Comment:
seems like rip and FCW/FTW can also be member initialized too

Reply:
Member initialized.